src/components/GoodsList.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const GoodsList = ({ goodsList,
+  getALlGoods,
+  getFirstFiveGoods,
+  getRedGoods }) => (
+    <>
+      <button
```

Comment:
All the controls should be in the `App` component. 
`GoodsList` should only render the list it receives as prop.